prime-router/src/main/kotlin/transport/GAENTransport.kt

```diff
@@ -240,12 +240,14 @@ class GAENTransport : ITransport, Logging {
         } else {
             // The follow error table is based on ENCV server docs.
             val postResult = when (response.statusCode) {
-                // Note: The majority of GAEN errors are invalid date content errors that can not be rectified by
-                // resubmission via the Last Mile Failures page. We have decided to ignore the 400 error code for
-                // GAEN messages as we are not taking action on these errors. This does not affect ELR messages
-                // sent to STLTs. STLTs will handle the content error and request for resubmission.
+                // Note: GAEN errors (Bad parameters, Unsupported test type, and UUID already present)
+                // can not be rectified by resubmission via the Last Mile Failures page. We have decided
+                // to ignore the 400 error code for GAEN messages as we are not taking action on
+                // these errors. This does not affect ELR messages sent to STLTs.
+                // STLTs will handle the content error and request for resubmission.
                 // Then, the new GAEN message from regenerate and resend accordingly.
-                400, 409 -> PostResult.SUCCESS // Bad parameters and UUID already present (consider this a success)
+                400, 409, 412 -> PostResult.SUCCESS // Bad parameters, Unsupported test type, and UUID already present
+                                                    // can not be rectified by re (consider this a success)
                 412 -> PostResult.FAIL // Unsupported test type
```

Comment:
If we are considering 412  a success, shouldn't we remove this?

Reply:
@GilmoreA6 no we can't remove it.  If we remove it, it will go to else and return PostResult.FAIL instead.